src/errorreport/service/ErrorReportingService.ts

```diff
@@ -0,0 +1,33 @@
+import SqsMessageService from '../../aws/sqs/service/SqsMessageService';
+import { Logger, LogLevel } from 'channelape-logger';
+
+import ErrorReport from '../model/ErrorReport';
+import AwsCredentials from '../../aws/model/AwsCredentials';
+
+export default class ErrorReportingService {
+  private readonly sqsMessageService: SqsMessageService;
+  private readonly logger: Logger;
+
+  constructor(awsCredentials: AwsCredentials, awsErrorsQueueUrl: string, logLevel: LogLevel | string) {
+    this.logger = new Logger('Error Reporting Service', logLevel);
+    this.sqsMessageService = new SqsMessageService(
+      awsCredentials.secretKey,
+      awsCredentials.accessKeyId,
+      awsErrorsQueueUrl,
+      awsCredentials.region
+    );
+  }
+
+  public queueError(params: ErrorReport): void {
+    const message = {
+      module: params.module,
+      channelOrderId: params.channelOrderId,
+      poNumber: params.poNumber,
+      channelApeOrder: `https://app.channelape.com/orders/${params.channelApeOrderId}`,
```

Comment:
@rjdavis3 Do you think it would be better to check for an environment variable instead?

```javascript
if (process.env.CHANNEL_APE_DOMAIN_NAME !== undefined) {
   channelApeDomainName = `https://${process.env.CHANNEL_APE_DOMAIN_NAME}/
}
```

Reply:
@frankMazzarella no I don't think you should add another env var for this just yet. Stick with the boolean for now. I think another env var like CHANNEL_APE_DOMAIN_NAME might confuse the App and the API.

@csimko  might have some feedback though. I know he had to add this logic into a past EI for error reporting.